### PR TITLE
vendor: remove coredns and ingress controller from k8scloudconfig

### PR DESCRIPTION
**Pull request by rossf7**
Towards giantswarm/giantswarm#4833

### CODE REVIEW

**Comment by rossf7** [comment]
Tested on ginger.

File: service/controller/v21/cloudconfig/master_template.go [hunk 1]
```diff
@@ -44,9 +44,7 @@ func (c *CloudConfig) NewMasterTemplate(ctx context.Context, customObject v1alph
 		params = k8scloudconfig.DefaultParams()
 
 		params.Cluster = customObject.Spec.Cluster
-		params.DisableCoreDNS = true
 		params.DisableEncryptionAtREST = true
-		params.DisableIngressController = true
```
Comment (rossf7) [inline service/controller/v21/cloudconfig/master_template.go]: Only non-vendor change is removing k8scc params. Already tested manually so will merge once e2e is green.

**Comment by kopiczko** [comment]
Can this be merged?

**Comment by rossf7** [comment]
@kopiczko Yes just back from lunch. Sry about the hold up.

**Pull request merged**